client/src/featureform/dashboard_metadata.py

```diff
@@ -118,8 +120,34 @@ def source_variant(variantData):
     allVariantList = []
     variants = []
     for variantRow in variantData:
+        try:
+            feature_list = sqlObject.get_feature_variants_from_source(variantRow['name'], variantRow['variant'])
+        except ValueError:
```

Comment:
is value error only triggered if there is no data? If its not a genuine error is there another way to single this? If it is a genuine error there is no error handling

Reply:
The value error is only triggered if data = []